modules/installation-network-user-infra.adoc

```diff
@@ -111,3 +111,12 @@ A working configuration for the Ingress router is required for an
 {product-title} cluster. You must configure the Ingress router after the control
 plane initializes.
 ====
+
+[discrete]
+== Ethernet adaptor hardware address requirements
+
+When provisioning VMs for the cluster, the ethernet interfaces configured for
```

Comment:
@dav1x, does this sound right to you, and would it be more meaningful in the section about creating VMs instead of setting up the network interfaces? (If it's more meaningful here, I think we need to clarify if the ethernet interfaces are really for each VM or if they're for a larger component.)

Reply:
This sounds right.